huxley/www/js/components/ChairSummaryView.js

```diff
@@ -187,10 +187,14 @@ var ChairSummaryView = React.createClass({
     var committee = CurrentUserStore.getCurrentUser().committee;
     var delegates = this.state.delegates;
     var summaries = this.state.summaries;
+    var toSave = [];
     for (var delegate of delegates) {
-      delegate.summary = summaries[delegate.assignment];
+      if (delegate.summary != summaries[delegate.assignment]) {
+        delegate.summary = summaries[delegate.assignment];
```

Comment:
Not a huge fan of directly mutating the state object. Do we use `delegate.summary` in the UI, or is that what `summaries` is for? If we don't, we can let this slide, but we should be working on making that separate model and moving summaries/attendance to it.